### DISCUSSION TITLE
The CustomResourceDefinition "kafkas.kafka.strimzi.io" is invalid

### DISCUSSION THREAD

**Discussion by Zodor** [strimzi/strimzi-kafka-operator#5093] [Q&A]
When applying the file: crds/040-Crd-kafka.yaml I am getting the following error:

`The CustomResourceDefinition "kafkas.kafka.strimzi.io" is invalid: metadata.annotations: Too long: must have at most 262144 bytes`

How to overcome this behaviour?

**Comment by scholzj** [comment]
In Strimzi 0.22, the Kafka CR is relatively big since it contains multiple API versions. You probably used `kubectl apply` to create it. But `kubectl apply` has some limitations to how big the resources can be which is the error you got there. You need to use `kubectl create` / `kubectl replace`. From 0.23 on, we removed the older API versions and `kubectl apply` should work again.

**Reply by Zodor** [reply]
We are running this from ArgoCD and the only way to deploy this is to "FORCE" deploy the cluster.
If an auto-sync is running, the FORCE is not used, so I have to resync manually every time the repo changes with the FORCE flag checked. This is very annoying.
